deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/K8sAuthenticationHandler.java

```diff
@@ -41,11 +45,13 @@ public boolean authenticate(AuthConfig authConfig) throws HyscaleException {
 		AuthenticationV1Api apiInstance = new AuthenticationV1Api(apiClient);
 		try {
 			V1APIResourceList result = apiInstance.getAPIResources();
-			return true;
+			return result != null ? true : false;
```

Comment:
What is the response in case of unauthorised access? Does it throw exception, returns null, or returns some response?